i18n/converter.py

```diff
@@ -26,6 +26,7 @@ class Converter(object):
         (<[^>]+>)           |       # <tag>
         ({[^}]+})           |       # {tag}
         (%\([\w]+\)\w)      |       # %(tag)s
+        (%[sdrxf])          |       # %s
```

Comment:
so you don't think anyone is going to have to specify an octal? :grin: 

actually I just looked it up, there are a ton of options that could be used here - better to encourage use of .format than support crazy %+f, %X, etc nonsense.